### PR TITLE
update description and links

### PR DESCRIPTION
**Pull request by jlovejoy**
update description and links to be current

### CODE REVIEW

**Comment by jlovejoy** [comment]
@swinslow - please have a look!

File: resources/htmlTemplate/ExceptionsTocHTMLTemplate.html [hunk 1]
```diff
@@ -135,14 +135,17 @@ <h1>License Exceptions</h1>
...
+	      <li style="margin:5px 0 5px 10px;"><a href="https://github.com/spdx/license-list-XML/blob/master/DOCS/license-fields.md"><strong>Explanation of fields</strong> used on the SPDX License List</li>
+	      <li style="margin:5px 0 5px 10px;"><a href="https://github.com/spdx/license-list-XML/blob/master/DOCS/license-inclusion-principles.md"><strong>License inclusion principles</strong> for adding new licenses or exceptions to the SPDX License List</li>
+	      <li style="margin:5px 0 5px 10px;"><a href="https://github.com/spdx/license-list-XML/blob/master/CONTRIBUTING.md"><strong>Contribute</strong> to the project or request a new license</li>
+	      <li style="margin:0px 0 5px 10px;">Use <a href="http://spdx.org/ids"><strong>short identifiers in your source code</strong></a></li>
+	      <li style="margin:5px 0 5px 10px;"><a href="https://github.com/spdx/license-list-XML"><strong>Github repo</strong></a></li>
+            <li style="margin:5px 0 5px 10px;">Machine readable <a href="https://github.com/spdx/license-list-data"><strong>data files</strong></a> for the SPDX License List</li>
 
       </ul>
+      </ul>
```
Comment (swinslow) [inline resources/htmlTemplate/ExceptionsTocHTMLTemplate.html:148]: I think this additional `</ul>` is duplicative and shouldn't be added, unless I'm misreading?

File: resources/htmlTemplate/ExceptionsTocHTMLTemplate.html [hunk 2]
```diff
@@ -135,14 +135,17 @@ <h1>License Exceptions</h1>
...
+            <li style="margin:5px 0 5px 10px;">The <a href="http://spdx.org/spdx-license-list/matching-guidelines"><strong>matching guidelines</strong></a> define what constitutes a license or exception match. The license text on the HTML pages here will display omitable text in blue and replaceable text in red (see Guideline #2 for more information).</li>
+	      <li style="margin:5px 0 5px 10px;"><a href="https://github.com/spdx/license-list-XML/blob/master/DOCS/license-fields.md"><strong>Explanation of fields</strong> used on the SPDX License List</li>
+	      <li style="margin:5px 0 5px 10px;"><a href="https://github.com/spdx/license-list-XML/blob/master/DOCS/license-inclusion-principles.md"><strong>License inclusion principles</strong> for adding new licenses or exceptions to the SPDX License List</li>
+	      <li style="margin:5px 0 5px 10px;"><a href="https://github.com/spdx/license-list-XML/blob/master/CONTRIBUTING.md"><strong>Contribute</strong> to the project or request a new license</li>
+	      <li style="margin:0px 0 5px 10px;">Use <a href="http://spdx.org/ids"><strong>short identifiers in your source code</strong></a></li>
```
Comment (swinslow) [inline resources/htmlTemplate/ExceptionsTocHTMLTemplate.html:143]: The URL here can be updated to https://spdx.dev/ids

File: resources/htmlTemplate/ExceptionsTocHTMLTemplate.html [hunk 3]
```diff
@@ -135,14 +135,17 @@ <h1>License Exceptions</h1>
...
-            <li style="margin:5px 0 5px 10px;"><a href="http://spdx.org/spdx-license-list/matching-guidelines"><strong>Matching Guidelines</strong></a>:  Guidelines for what constitutes a license match to the SPDX License List. For licenses that include markup, the license text on the HTML pages here will display omitable text in blue and replaceable text in red (see Guideline #2 for more information).</li>
-            <li style="margin:5px 0 5px 10px;"><a href="https://github.com/spdx/license-list-XML/blob/master/CONTRIBUTING.md"><strong>Request New License</strong></a>: Instructions for proposing a license or exception be added to the SPDX License List.</li>
+            <li style="margin:5px 0 5px 10px;"><a href="index.html"><strong>SPDX License List</strong></a></li>
+            <li style="margin:5px 0 5px 10px;">The <a href="http://spdx.org/spdx-license-list/matching-guidelines"><strong>matching guidelines</strong></a> define what constitutes a license or exception match. The license text on the HTML pages here will display omitable text in blue and replaceable text in red (see Guideline #2 for more information).</li>
```
Comment (swinslow) [inline resources/htmlTemplate/ExceptionsTocHTMLTemplate.html:139]: The URL here can be updated to https://spdx.dev/license-list/matching-guidelines/

File: resources/htmlTemplate/TocHTMLTemplate.html [hunk 4]
```diff
@@ -136,21 +136,19 @@ <h1><a href="/" title="Home" rel="home">Software Package Data Exchange (SPDX)</a
...
+            <li style="margin:5px 0 5px 10px;">The <a href="http://spdx.org/spdx-license-list/matching-guidelines"><strong>matching guidelines</strong></a> define what constitutes a license or exception match. The license text on the HTML pages here will display omitable text in blue and replaceable text in red (see Guideline #2 for more information).</li>
+	      <li style="margin:5px 0 5px 10px;"><a href="https://github.com/spdx/license-list-XML/blob/master/DOCS/license-fields.md"><strong>Explanation of fields</strong> used on the SPDX License List</li>
+	      <li style="margin:5px 0 5px 10px;"><a href="https://github.com/spdx/license-list-XML/blob/master/DOCS/license-inclusion-principles.md"><strong>License inclusion principles</strong> for adding new licenses or exceptions to the SPDX License List</li>
+	      <li style="margin:5px 0 5px 10px;"><a href="https://github.com/spdx/license-list-XML/blob/master/CONTRIBUTING.md"><strong>Contribute</strong> to the project or request a new license</li>
+	      <li style="margin:0px 0 5px 10px;">Use <a href="http://spdx.org/ids"><strong>short identifiers in your source code</strong></a></li>
```
Comment (swinslow) [inline resources/htmlTemplate/TocHTMLTemplate.html:149]: The URL can be updated to https://spdx.dev/ids

File: resources/htmlTemplate/TocHTMLTemplate.html [hunk 5]
```diff
@@ -136,21 +136,19 @@ <h1><a href="/" title="Home" rel="home">Software Package Data Exchange (SPDX)</a
...
-            <li style="margin:5px 0 5px 10px;"><a href="https://github.com/spdx/license-list-XML/blob/master/CONTRIBUTING.md"><strong>Request New License</strong></a>: Instructions for proposing a license or exception be added to the SPDX License List.</li>
+            <li style="margin:5px 0 5px 10px;"><a href="./exceptions-index.html"><strong>License Exceptions</strong></a> are commonly found exceptions to free and open source licenses, used with the <strong><a href="https://spdx.org/specifications">License Expression</a></strong> operator, "WITH" to create a license with an exception. </li>
+            <li style="margin:5px 0 5px 10px;">The <a href="http://spdx.org/spdx-license-list/matching-guidelines"><strong>matching guidelines</strong></a> define what constitutes a license or exception match. The license text on the HTML pages here will display omitable text in blue and replaceable text in red (see Guideline #2 for more information).</li>
```
Comment (swinslow) [inline resources/htmlTemplate/TocHTMLTemplate.html:145]: The URL can be updated to https://spdx.dev/license-list/matching-guidelines/

File: resources/htmlTemplate/TocHTMLTemplate.html [hunk 6]
```diff
@@ -136,21 +136,19 @@ <h1><a href="/" title="Home" rel="home">Software Package Data Exchange (SPDX)</a
...
-            <li style="margin:0px 0 5px 10px;"><a href="http://spdx.org/ids"><strong>IDs</strong></a>: Learn more about using short-form license identifiers in your source code.</li>
-            <li style="margin:5px 0 5px 10px;"><a href="http://spdx.org/spdx-license-list/matching-guidelines"><strong>Matching Guidelines</strong></a>:  Guidelines for what constitutes a license match to the SPDX License List. The license text on the HTML pages here will display omitable text in blue and replaceable text in red (see Guideline #2 for more information).</li>
-            <li style="margin:5px 0 5px 10px;"><a href="https://github.com/spdx/license-list-XML/blob/master/CONTRIBUTING.md"><strong>Request New License</strong></a>: Instructions for proposing a license or exception be added to the SPDX License List.</li>
+            <li style="margin:5px 0 5px 10px;"><a href="./exceptions-index.html"><strong>License Exceptions</strong></a> are commonly found exceptions to free and open source licenses, used with the <strong><a href="https://spdx.org/specifications">License Expression</a></strong> operator, "WITH" to create a license with an exception. </li>
```
Comment (swinslow) [inline resources/htmlTemplate/TocHTMLTemplate.html:144]: The second URL (for the specifications link) can be updated to https://spdx.dev/specifications/

File: resources/htmlTemplate/TocHTMLTemplate.html [hunk 7]
```diff
@@ -136,21 +136,19 @@ <h1><a href="/" title="Home" rel="home">Software Package Data Exchange (SPDX)</a
...
+            <li style="margin:5px 0 5px 10px;">The <a href="http://spdx.org/spdx-license-list/matching-guidelines"><strong>matching guidelines</strong></a> define what constitutes a license or exception match. The license text on the HTML pages here will display omitable text in blue and replaceable text in red (see Guideline #2 for more information).</li>
+	      <li style="margin:5px 0 5px 10px;"><a href="https://github.com/spdx/license-list-XML/blob/master/DOCS/license-fields.md"><strong>Explanation of fields</strong> used on the SPDX License List</li>
```
Comment (swinslow) [inline resources/htmlTemplate/TocHTMLTemplate.html:146]: Here and below, some of the links are missing the closing `</a>` tag outside the closing `</strong>` tag.

File: resources/htmlTemplate/ExceptionsTocHTMLTemplate.html [hunk 8]
```diff
@@ -135,14 +135,17 @@ <h1>License Exceptions</h1>
...
+	      <li style="margin:5px 0 5px 10px;"><a href="https://github.com/spdx/license-list-XML/blob/master/DOCS/license-fields.md"><strong>Explanation of fields</strong> used on the SPDX License List</li>
```
Comment (swinslow) [inline resources/htmlTemplate/ExceptionsTocHTMLTemplate.html:140]: Here and below, some of the links are missing the closing `</a>` tag outside the closing `</strong>` tag.

**Review by swinslow — CHANGES_REQUESTED** [review]
Hi @jlovejoy, the substance of this looks good to me. There are a few necessary formatting edits (missing closing `</a>` tags to be added and possibly an extra `</ul>` tag to be deleted), and a few less-necessary recommended edits to update URLs. 

I'm making changes to the version that I'm implementing on the live site for the 3.11 release to address these comments, and I will circle back to update them in this PR -- or feel free to make the edits here yourself if you have time. Thanks!

**Comment by goneall** [comment]
@jlovejoy @swinslow There is a change request on this PR - if you could review/update I can merge in the changes before the next license list release.

**Review by swinslow — APPROVED** [review]

**Comment by swinslow** [comment]
Thanks @goneall, I've pushed the changes to fix the issues I flagged above -- should be good to go now.

**Comment by goneall** [comment]
Thanks @swinslow !

**Pull request merged**